apps/test/unit/templates/curriculumCatalog/curriculumCatalogExpandedCardTest.jsx

```diff
@@ -0,0 +1,256 @@
+      imageSrc:
+        'https://images.code.org/58cc5271d85e017cf5030ea510ae2715-AI for Oceans.png',
+    };
+  });
+
+  afterEach(() => {
+    restoreRedux();
+  });
+
+  it('renders course name', () => {
+    renderCurriculumExpandedCard();
+
+    screen.getByRole('heading', {name: defaultProps.courseDisplayName});
+  });
+
+  it('renders grade range with icon', () => {
+    const {container} = renderCurriculumExpandedCard();
+
+    screen.getByText(defaultProps.gradeRange);
+    expect(container.querySelectorAll('i[class*=user]')).to.have.length(1);
+  });
+
+  it('renders single grade with icon when one grade passed in', () => {
+    const grade = 'Grades: 12';
+    const {container} = renderCurriculumExpandedCard({
+      ...defaultProps,
+      gradeRange: grade,
+    });
+
+    screen.getByText(grade);
+    expect(container.querySelectorAll('i[class*=user]')).to.have.length(1);
+  });
+
+  it('renders duration with icon', () => {
+    const duration = 'quarter';
+    const {container} = renderCurriculumExpandedCard({
+      ...defaultProps,
+      duration: duration,
+    });
+
+    screen.getByText(duration);
+    expect(container.querySelectorAll('i[class*=clock]')).to.have.length(1);
+  });
+
+  it('renders topics with icon', () => {
+    const {container} = renderCurriculumExpandedCard();
+
+    screen.getByText(
+      new RegExp(`Topic: ${defaultProps.subjectsAndTopics.join(', ')}`)
+    );
+    expect(container.querySelectorAll('i[class*=book]')).to.have.length(1);
+  });
+
+  it('renders video if available', () => {
+    renderCurriculumExpandedCard();
+
+    screen.getAllByTitle('Youtube embed');
+  });
+
+  it('renders image if no video available', () => {
+    renderCurriculumExpandedCard({
+      ...defaultProps,
+      video: null,
+    });
+
+    screen.getByRole('img');
+  });
+
+  it('renders image with alt text if passed and no video', () => {
+    const altText = 'Ocean and fishes';
+    renderCurriculumExpandedCard({
+      ...defaultProps,
+      video: null,
+      imageAltText: altText,
+    });
+
+    // when alt text is present, the img name is the alt text
+    screen.getByRole('img', {name: altText});
+  });
+
+  it('renders available resources section when resources are available', () => {
+    const availableResources = {
+      'Lesson Plan': '/s/oceans/lessons/1',
+      'Slide Deck': 'slides',
+    };
+    renderCurriculumExpandedCard({
+      ...defaultProps,
+      availableResources: availableResources,
+    });
+
+    const availableResourcesContainer = screen.getByText(
+      'Available Resources'
+    ).parentElement;
+
+    //Checks for each available resource in availableResources
+    Object.keys(availableResources).forEach(resource => {
+      screen.getByText(translatedAvailableResources[resource]);
+    });
+    //Checks for correct amount of Horizontal dividers
+    expect(availableResourcesContainer.querySelectorAll('hr')).to.have.length(
+      Object.keys(availableResources).length
```

Comment:
Shouldn't there be 1 fewer dividers than number of resources since the dividers go between the resources?

Reply:
This is also making sure the divider under available resources is there essentially.